File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/constants/constantsForInputTypes/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.constants.constantsForInputTypes.expected;

import java.lang.String;

public class DgsConstants {
  public static final String QUERY_TYPE = "Query";

  public static class QUERY {
    public static final String TYPE_NAME = "Query";

    public static final String People = "people";

    public static class PEOPLE_INPUT_ARGUMENT {
      public static final String Filter = "filter";
    }
  }

  public static class PERSON {
    public static final String TYPE_NAME = "Person";

    public static final String Firstname = "firstname";

    public static final String Lastname = "lastname";
  }

  public static class PERSONFILTER {
    public static final String TYPE_NAME = "PersonFilter";

    public static final String Email = "email";
  }
}

File: graphql-dgs-codegen-core/src/integTest/kotlin/com/netflix/graphql/dgs/codegen/cases/constantsForInputTypes/expected/DgsConstants.kt
package com.netflix.graphql.dgs.codegen.cases.constantsForInputTypes.expected

import kotlin.String

public object DgsConstants {
  public const val QUERY_TYPE: String = "Query"

  public object PERSON {
    public const val TYPE_NAME: String = "Person"

    public const val Firstname: String = "firstname"

    public const val Lastname: String = "lastname"
  }

  public object QUERY {
    public const val TYPE_NAME: String = "Query"

    public const val People: String = "people"

    public object PEOPLE_INPUT_ARGUMENT {
      public const val Filter: String = "filter"
    }
  }

  public object PERSONFILTER {
    public const val TYPE_NAME: String = "PersonFilter"

    public const val Email: String = "email"
  }
}
